src/renderer/components/screens/AccountDeletionScreen/styles.module.scss

```diff
@@ -0,0 +1,60 @@
+.AccountDeletionScreen {
+  flex-grow: 1;
+  .Background {
+    background-size: cover;
+    // TODO find a better fire/lava image
+    background-image: url('../images/account_deletion_background.jpg');
+    height: 100vh;
+    margin-top: 0px;
+  }
+}
+
+.avatar {
```

Comment:
Think this needs a `cursor: pointer`

Reply:
@r10s what do you say?

Reply:
I propose that anything which is clickable should be indicated to the user with a pointer cursor.

"Settings" and "Create Account" already have one, why not also for switching accounts?